psycopg/setup.cfg

```diff
@@ -5,6 +5,7 @@ url = https://psycopg.org/psycopg3/
 author = Daniele Varrazzo
 author_email = daniele.varrazzo@gmail.com
 license = GNU Lesser General Public License v3 (LGPLv3)
+version = 3.1.8.dev1
```

Comment:
It would be wise to release this change only in psycopg 3.2. I will open a "maint 3.1" branch as soon as we are ready to merge one of the new features branch. So I think we can set the version to `3.2.0.dev1` (here and in the `setup.cfg` dependencies).